android/java/org/chromium/chrome/browser/crypto_wallet/util/AccountsPermissionsHelper.java

```diff
@@ -45,8 +46,8 @@ public void checkAccounts(Runnable runWhenDone) {
 
             accountsPermissionsContexts.add(accountPermissionContext);
 
-            mBraveWalletService.hasEthereumPermission(
-                    mOrigin, account.address, accountPermissionContext);
+            mBraveWalletService.hasPermission(
```

Comment:
cc @SergeyZhukovsky for related android change

Reply:
java changes looks ok